frontend/src/components/reviews/Review.tsx

```diff
@@ -28,10 +34,12 @@ const Review = ({
             {new Date(review.submittedAt).toLocaleString("en-US")}
           </Text>
         </Group>
+      ) : (
+        <div><MovieInfo id={review.movieId}/></div>
```

Comment:
The review is meant to be shown on the main movie page with all the data, I would create a SmallReviewComponent instead to show on profiles